apps/article/urls.py

```diff
@@ -8,7 +8,7 @@
 urlpatterns = [
     url(r'^archive/$', views.archive, name='article_archive'),
     url(r'^(?P<article_id>\d+)/(?P<article_slug>[a-zA-Z0-9_-]+)/$', views.details, name='article_details'),
-    url(r'^tag/(?P<slug>[^\.]+)/', views.archive_tag, name='view_article_tag'),
```

Comment:
I don't think this URL needs to change its name, it's not related to the permissions.